global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "6.0.100-rc.1.21430.12",
+    "dotnet": "6.0.100-rc.1.21458.32",
```

Comment:
this will need to be reverted

Reply:
(seperate PR for unrelated changes)

Reply:
@joperezr this seems to be making the CI green again

Reply:
I revert back and it is successfully passed